### PR TITLE
Adds a lesson about fixing PostgreSQL write performance.

### PR DESCRIPTION
**Pull request by jakubpawlowicz**

### CODE REVIEW

**Comment by omartell** [comment]
:+1:

File: postgresql/write_performance.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+We have a write-heavy application which needs to keep up with data streamed from an external system. Apparently our staging environment wasn't able to cope with writing all the data to a PostgreSQL database, so our data backlog grew larger and larger.
+
+We quickly verified it's not a memory- or a CPU-bound issue via `top`. However [iotop](http://guichaz.free.fr/iotop/) showed an unusually high number of I/O operations by `jbd2/md6-8` process:
+
+```
+TID  PRIO  USER     DISK READ  DISK WRITE  SWAPIN     IO>    COMMAND
+477  be/3  root      0.00 B/s  2778.70 K/s  0.00 % 67.48 % [jbd2/md6-8]
+```
+
+The `jbd2` process manages journalling on [ext4](https://ext4.wiki.kernel.org/index.php/Ext4_Howto) filesystem, which shouldn't clearly be running on a partition where PostgreSQL keeps its data. That's because PostgreSQL keeps its own journaling in terms of [Write-Ahead Log](http://www.postgresql.org/docs/9.4/static/wal-intro.html).
```
Comment (mudge) [inline postgresql/write_performance.md]: "clearly shouldn't" instead?

Comment (jakubpawlowicz) [inline postgresql/write_performance.md]: :+1:

Comment (jakubpawlowicz) [inline postgresql/write_performance.md]: Fixed.

Comment (scottmatthewman) [inline postgresql/write_performance.md]: s/filesystem/filesystems/

**Comment by scottmatthewman** [comment]
I'd also have suggested changing "clearly shouldn't" to something more like "is unnecessary", or "conflicts with Postgres's own journalling".

Terms like "clearly" and "just" are value judgements about the ease of a task, which are not only highly subjective but can put barriers up for other people as they are learning. And more often than not, in a TIL context they're not true – if things really were that clear, they may not have prompted the writing of a TIL in the first place :smile:

**Comment by jakubpawlowicz** [comment]
Thanks a lot @scottmatthewman for a lesson on how to write a good TIL. I'll send another PR with changes.

**Pull request merged**